docs/src/baby_fuzzer.md

```diff
@@ -178,7 +178,7 @@ state
 
 Now you can prepend the necessary `use` directives to your main.rs and compile the fuzzer.
 
-```rust 
+```rust,ignore
```

Comment:
why do we need to ignore this?

Reply:
the books fails without `ignore` https://github.com/AFLplusplus/LibAFL/runs/4389266826

Reply:
I don't know what it is going on internally.... though

Reply:
Then we should not run the test on windows, but not completely disable the tests

Reply:
I don't understand why the ci is green on the main branch.
This command `mdbook test -L ../target/debug/deps` fails even on ubuntu on _main_ branch.
and I see very different error message on both windows and linux compared to what CI says.
On my machine it says,
```
 --> C:\Users\toka\AppData\Local\Temp\mdbook-vSkmfI\baby_fuzzer.md:78:1
  |
2 | extern crate libafl;
  | ^^^^^^^^^^^^^^^^^^^^ can't find crate
```

I think we need this ignore...

Reply:
Did you `cargo build` before? Then it should find the dependencies(but it won't build then by itself)
Also, are you in a different folder, maybe?

Reply:
Obviously the current test works else CI shouldn't be green 😛

Reply:
> Did you cargo build before? Then it should find the dependencies(but it won't build then by itself)
No, I didn't, will look at this tomorrow

Reply:
it works on linux :)
so I have to figure out why msvc compiler tries to link to this nonexistent 'windows.lib' file

Reply:
mm
I'll change version back to windows:0.18 if I can't find an immediate solution. (and possibly do another windows bump pr to separate the problem, another pr is dependent on this pr, so it kinda is getting messy)

Reply:
Why not leave the new version and skip the docu build step for windows for now?